infrastructure/index.ts

```diff
@@ -81,6 +80,29 @@ const uploadsBucket = new aws.s3.Bucket("uploads-bucket", {
     }],
 });
 
+// This needs to be set in order to allow the use of ACLs. This was added to update our infrastructure to be
+// compatible with the default S3 settings from AWS' April update. `ObjectWriter` was the prior default, so
+// changing it to that here to match the configuration prior to the update.
+// https://aws.amazon.com/blogs/aws/heads-up-amazon-s3-security-changes-are-coming-in-april-of-2023/
+const uploadsBucketOwnershipControls = new aws.s3.BucketOwnershipControls("uploads-bucket-ownership-controls", {
+    bucket: uploadsBucket.id,
+    rule: {
+        objectOwnership: "ObjectWriter"
+    }
+});
+
+const uploadsBucketPublicAccessBlock = new aws.s3.BucketPublicAccessBlock("uploads-public-access-block", {
+    bucket: uploadsBucket.id,
+    blockPublicAcls: false,
+});
+
+const uploadsBucketAcl = new aws.s3.BucketAclV2("uploads-bucket-acl", {
```

Comment:
@zchase, I don't _think_ it is needed.  My intention was just to get parity with the configuration that existed previously so that we match things as close as possible to how they were before, since before this we explicitly had a public-read acl on the bucket.